conf/gulp.conf.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
```

Comment:
is there a specific reason you removed this?

Reply:
Not really. I never use it, but it might be best practice to.